lutris/runners/commands/wine.py

```diff
@@ -223,6 +226,7 @@ def wineexec(  # noqa: C901
         blocking (bool): if true, do not run the process in a thread
         config (LutrisConfig): LutrisConfig object for the process context
         watch (list): list of process names to monitor (even when in a ignore list)
+        runner (runner): the wine runner that carries teh configuration to use
```

Comment:
```suggestion
        runner (runner): the wine runner that carries the configuration to use
```
Small typo ;)

Reply:
Thank you. Though I'm tempted to leave it in there to see what the localization guys make of it.